dataproc/quickstart/quickstart.go

```diff
@@ -181,41 +191,27 @@ func main() {
 
 	resp, err := clusterClient.GetCluster(ctx, getCReq)
 	if err != nil {
-		fmt.Errorf("error getting cluster %q: %v", clusterName, err)
+		fmt.Println(fmt.Errorf("error getting cluster %q: %v", clusterName, err))
```

Comment:
No need to create an error then print it. Can just print directly. Return after this (and the other errors)?